packages/functional-tests/tests/settings/recoveryKey.spec.ts

```diff
@@ -374,6 +376,8 @@ test.describe('new recovery key test', () => {
     page,
     pages: { settings, recoveryKey, login },
   }) => {
+    // TODO: Possibly duplicate coverage in /react-conversion/recoveryKey.spec.ts
```

Comment:
Woud be nice to have follow up to combine/remove all the duplicate tests.

Reply:
Filed FXA-8267